NugetAuditor/AuditResult.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) 2015-2018, Sonatype Inc.
+// All rights reserved.
+// 
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Sonatype, OSS Index, nor the
+//       names of its contributors may be used to endorse or promote products
+//       derived from this software without specific prior written permission.
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+// ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL SONATYPE BE LIABLE FOR ANY
+// DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+// LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+// ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+using NugetAuditor.Lib.OSSIndex;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NugetAuditor.Lib
+{
+    public class AuditResult
+    {
+        private PackageId _packageId;
+        private Package _package;
+        
+        public PackageId PackageId
```

Comment:
Seems like there are a lot of formatting errors here. Make sure your editor is set to either tabs OR spaces, looks like there's a mix of both.

Reply:
Running Resharper across the whole solution made a slew of changes it appears the inconsistencies in line endings mostly came from the lib I pulled from Audit.Net, but there were several indentation inconsistencies (various spacing counts on indentation) throughout SafeNuGet as well so it hit a lot of files.

Reply:
It is quite a massive set of changes, so maybe a good time to make code style consistent. Not my call, I'm a humble/interested observer, of course.